### PR TITLE
Password expiry additions

### PR DESCRIPTION
**Pull request by pbdeuchler**
Add PasswordExpirationDate field to JCUser struct and passwordExpiryWatcher script

### CODE REVIEW

**Comment by pbdeuchler** [comment]
Output of running `passwordExpiryWatcher.go`:

screen shot 2016-06-24 at 2 34 15 pm

File: jcapi-systemusers.go [hunk 1]
```diff
@@ -138,9 +141,18 @@ func getJCUserFieldsFromInterface(fields map[string]interface{}, user *JCUser) {
...
+		user.PasswordExpirationDate, err = time.Parse(time.RFC3339, fields["password_expiration_date"].(string))
+		if err != nil {
+			return err
+		}
+	}
+	return nil
 }
 
-func getJCUsersFromInterface(userInt interface{}) []JCUser {
+func getJCUsersFromInterface(userInt interface{}) ([]JCUser, error) {
```
Comment (kusold) [inline jcapi-systemusers.go:155]: This signature change probably breaks a lot of the other examples. We need to update those. Same with function above

Comment (pbdeuchler) [inline jcapi-systemusers.go:155]: Yeah I thought so too, but luckily it's a private function so the examples shouldn't be able to use it

Comment (kusold) [inline jcapi-systemusers.go:155]: Ah. Missed that.

**Comment by kusold** [comment]
:+1:

**Comment by beckjake** [comment]
:+1:

**Pull request merged**